src/components/pacbio/PacbioRunInfoEdit.vue

```diff
@@ -110,6 +110,9 @@ export default {
     smrtLinkVersionv12() {
       return /^v12/.test(this.smrtLinkVersion.name)
     },
+    isRevio() {
```

Comment:
Probably best to do === adds an extra level of checking.

Reply:
Ok 👍